src/Build/project.json

```diff
@@ -22,6 +23,7 @@
         "System.Reflection.Metadata": "1.3.0",
         "System.Reflection.TypeExtensions": "4.1.0",
         "System.Runtime": "4.1.0",
+        "System.Runtime.InteropServices.RuntimeInformation": "4.0.0",
```

Comment:
Honestly I envisioned spending another day or so on this if I changed the .net core version, so decided to leave it alone :)
I'll give it a try and diff the lock files to see what happens.